api/v1/tigerastatus_types.go

```diff
@@ -113,7 +113,7 @@ type TigeraStatusList struct {
 	Items           []TigeraStatus `json:"items"`
 }
 
-// TigeraStatusReason represents the reason for a particular condition. A reason may be one of the below.
+// TigeraStatusReason represents the reason for a particular condition. The value should be a CamelCase string.
```

Comment:
I don't think this new comment or the one it replaced provides any help to the end user. This new comment seems to be more for someone adding or changing the code rather than an end user.

Reply:
For now I think it is best to simply remove the second sentence, such that the generated documentation makes sense.

Reply:
Update the comment.